src/components/MoneyRequestConfirmationList.js

```diff
@@ -481,10 +490,11 @@ function MoneyRequestConfirmationList(props) {
                     <ConfirmedRoute transactionID={props.transactionID} />
                 </View>
             )}
-            {!_.isEmpty(props.receiptPath) ? (
+            {receiptImage || receiptThumbnail ? (
                 <Image
                     style={styles.moneyRequestImage}
-                    source={{uri: ReceiptUtils.getThumbnailAndImageURIs(props.receiptPath, props.receiptSource).image}}
+                    source={{uri: receiptThumbnail || receiptImage}}
+                    isAuthTokenRequired={!_.isEmpty(receiptThumbnail)}
```

Comment:
Did we make the changes above because we reuse this component in `SplitBillDetailsPage` and need an authToken once the image is uploaded? If so, could we keep the same logic and use something like:

```suggestion
                    isAuthTokenRequired={props.isReadOnly}
```

Reply:
I'm working next on allowing the split details page to be editable, which will make `isReadOnly=false` so I don't think this change would work?

>Did we make the changes above because we reuse this component in SplitBillDetailsPage and need an authToken once the image is uploaded? If so, could we keep the same logic and use something like:

Basically to display the thumbnail in the details page, OR the image when it's a `blob://` `file://` when first creating a request. What do you think?

Reply:
Got it. I think what we have is fine then

Reply:
Great thanks!